content/07.Publication_Process.md

```diff
@@ -1,15 +1,38 @@
 # The Publication and Peer Review Process 
```

Comment:
You may consider including how to deal with pre-registered studies. @DrMattG and @ErlendNilsen can provide some input on this.

Reply:
Okay, I will have a think about this thanks.